tests/TestHelpers/GraphHelper.php

```diff
@@ -26,14 +26,32 @@ private static function getRequestHeaders(): array {
 		];
 	}
 
+	/**
+	 *
+	 * @return string
+	 */
+	public static function getUUIDv4Regex(): string {
+		return '[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}';
```

Comment:
why returning string,,,the func. name says regex.
```suggestion
		return '/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/';
```
this way you can avoid string concatenation like in L43.

Reply:
Needed this to build other regex patterns like in L53 for space id